tests/integration/render.int.js

```diff
@@ -217,7 +217,32 @@ describe('Full dummy app render', () => {
 			};
 			return server
 				.inject(request)
+				.then(response => {
+					expect(response.statusCode).toBe(500);
+				})
+				.then(() => server.stop())
+				.catch(err => {
+					server.stop();
+					throw err;
+				});
+		}));
+	it('returns 200 OK after a 500 error', () =>
```

Comment:
I'll add a followup ticket to write a dedicated set of 'side effect integration tests' to make sure that errors in `NotFound`, `Forbidden`, and `Redirect` are also cleaned up correctly. Unfortunately there's not a clear way to automatically apply the cleanup to any future react-side-effect component, or write a generic test that will check all instances, so I'll need to write separate tests.